030-Data-types/030-data-model.mdx

```diff
@@ -205,29 +205,38 @@ In the above, it's worth noting:
 
 ## Column types
 
-### string
+### text
 
-The `string` type represents a simple string. The values of this type are indexed both for quick exact matching and for full-text search. Set `unique` to ensure the strings you insert are unique. Set `notNull` to require a value.
+The `text` type represents a string or a long-form text. If a value must be provided, you can use the `notNull` setting to enforce this requirement.
 
 Example definition:
 
 ```json
 {
-  "name": "name",
-  "type": "string"
+  "name": "address",
+  "type": "text"
 }
 ```
 
-### text
+<Alert status="warning">
+  Note: Uniqueness on text columns can only be enabled on Postgres-enabled databases via the UI.
```

Comment:
```suggestion
  Note: Uniqueness on text columns can only be enabled on Postgres-enabled databases via the CLI.
```

Reply:
@eemmiillyy I have reworded this section to make it more clear.